pkg/operator/ceph/webhook.go

```diff
@@ -30,35 +29,70 @@ import (
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/apimachinery/pkg/util/version"
 	"sigs.k8s.io/controller-runtime/pkg/webhook"
 )
 
 const (
 	admissionControllerAppName       = "rook-ceph-admission-controller"
 	tlsPort                    int32 = 443
+	minK8sVersionWebhook             = "1.16.0"
 )
 
 var (
-	namespace = os.Getenv(k8sutil.PodNamespaceEnvVar)
+	namespace              = os.Getenv(k8sutil.PodNamespaceEnvVar)
+	certManagerWebhookName = "cert-manager-webhook"
 )
 
 func isSecretPresent(ctx context.Context, context *clusterd.Context) (bool, error) {
```

Comment:
To feel more confidence in enabling this by default, I think we need an env var (e.g. `ROOK_DISABLE_ADMISSION_CONTROLLER`) on the operator.yaml that allows disabling it. Then if the webhook fails in some corner cases, users can disable it until we can release a fix. This method could look for that env var and return if true.